From af981ecdc690ea7041918b71406620f9577fdf55 Mon Sep 17 00:00:00 2001
From: Aalique Grahame <agrahame@codeaurora.org>
Date: Fri, 9 Sep 2016 10:29:39 -0700
Subject: [PATCH] markw: configs: add support for 32/48KHz voip

Add 32/48KHz sampling rates to 8998 config file
and 32KHz sampling rate to 8996 config file to
allow voip applications with 32/48KHz to route to
the voip path

CRs-Fixed: 1065380
Change-Id: I0801f912977d9aed783c33a308288eae422a851b
Signed-off-by: TheHungarianHorntail <sayan7848@gmail.com>
---
 audio/audio_policy_configuration.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/audio/audio_policy_configuration.xml b/audio/audio_policy_configuration.xml
index 5b2cae9..174ad4f 100644
--- a/audio/audio_policy_configuration.xml
+++ b/audio/audio_policy_configuration.xml
@@ -131,7 +131,7 @@
                 <mixPort name="voip_rx" role="source"
                          flags="AUDIO_OUTPUT_FLAG_DIRECT|AUDIO_OUTPUT_FLAG_VOIP_RX">
                     <profile name="" format="AUDIO_FORMAT_PCM_16_BIT"
-                             samplingRates="8000,16000" channelMasks="AUDIO_CHANNEL_OUT_MONO"/>
+                             samplingRates="8000,16000,32000" channelMasks="AUDIO_CHANNEL_OUT_MONO"/>
                 </mixPort>
 
                 <mixPort name="primary input" role="sink">
